.yarn/versions/a263f870.yml

```diff
@@ -0,0 +1,5 @@
+releases:
+  "@radix-ui/react-slider": minor
```

Comment:
Should have been minor in the previous slider PR really with the new `inverted` prop, but this fixes it anyway.